<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizDialog</class>
 <widget class="QDialog" name="QuizDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="lblCount"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="lblQuestion"/>
   </item>
   <item>
    <widget class="QLineEdit" name="edtAnswer"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>Next</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
